<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchDialog</class>
 <widget class="QDialog" name="SearchDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>324</width>
    <height>125</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>251</width>
     <height>62</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayoutMain">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Чт&amp;о:</string>
          </property>
          <property name="buddy">
           <cstring>txtSearch</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txtSearch"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox">
        <property name="text">
         <string>С у&amp;четом регистра</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QPushButton" name="btnSearch">
        <property name="text">
         <string>&amp;Найти далее</string>
        </property>
        <property name="default">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>Отмена</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>txtSearch</tabstop>
  <tabstop>checkBox</tabstop>
  <tabstop>btnSearch</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>SearchDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>239</x>
     <y>72</y>
    </hint>
    <hint type="destinationlabel">
     <x>202</x>
     <y>124</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
